src/vs/platform/actions/common/actions.ts

```diff
@@ -234,6 +234,7 @@ export class MenuId {
 	static readonly ChatEditingWidgetToolbar = new MenuId('ChatEditingWidgetToolbar');
 	static readonly ChatEditingEditorContent = new MenuId('ChatEditingEditorContent');
 	static readonly ChatEditingEditorHunk = new MenuId('ChatEditingEditorHunk');
+	static readonly ChatEditingDeletedNotebookCell = new MenuId('ChatEditingDeletedNotebookCell');
```

Comment:
Cloned the menu, as I do not want to hack it by providing custom args, as the original menu expects a specific context as the args.